src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalGATKReadFilterPluginArgumentCollection.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+
+import java.util.*;
+
+/**
+ * {@link GATKReadFilterPluginArgumentCollection} for optional read filters in the command line. It allows:
+ *
+ * - Provide a list of read filters to apply.
+ * - Disable some and/or all read filters.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public class OptionalGATKReadFilterPluginArgumentCollection extends GATKReadFilterPluginArgumentCollection {
+    private static final long serialVersionUID = 1L;
+
+    @Argument(fullName = StandardArgumentDefinitions.READ_FILTER_LONG_NAME,
+            shortName = StandardArgumentDefinitions.READ_FILTER_SHORT_NAME,
+            doc="Read filters to be applied before analysis", optional=true)
```

Comment:
Although I think the common attribute will go away in the near future, for now we should make all of these args consistent and have common=true.